src/lib/accessibility/Behaviors/Popup/popupFocusTrapBehavior.ts

```diff
@@ -0,0 +1,17 @@
+import { Accessibility } from '../../types'
+import popupBehavior from './popupBehavior'
+import * as _ from 'lodash'
+
+/**
+ * @description
+ * Adds role='button' to 'trigger' component's part, if it is not focusable element and no role attribute provided.
+ * Adds tabIndex='0' to 'trigger' component's part, if it is not tabbable element and no tabIndex attribute provided.
+ * Adds attribute 'aria-disabled=true' to 'trigger' component's part based on the property 'disabled'.
+ * Wraps component in FocusTrapZone, with default prop 'isClickableOutsideFocusTrap' set to 'true'.
+ */
+const popupFocusTrapBehavior: Accessibility = (props: any) => ({
```

Comment:
Yep, firstly we decided that the default one would be with focus trap. But after saw my initial commit, it had confusion in namings and understanding between behavior with focus trap and without. We agreed with @jurokapsiar on these options
- popupBehavior (without focus trap, default)
- popupFocusTrapBehavior (with focus trap)

So to bring both semantically descriptive name and more intuitive logic for the clients.
Everyone still can have focus trap, because we expose ```popupFocusTrapBehavior``` to the Stardust lib and show a miningful example on how to use it.

@kuzhelov if you have any other thoughts on it, please, feel free, to share.